doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -1206,13 +1219,14 @@ The value is the fetch interval in seconds for new OCSP responses.
-|-----------------------------------|------------------------------------------------------------------------------------------------------|--------------------------------------------------------:|
-| xroad-center                      |                                                 X-Road Central Server                                                     |        `/var/log/xroad/centralserver-admin-service.log` |
-| xroad-center-registration-service |                              X-Road Central Server Registration Service                              | `/var/log/xroad/centralserver-registration-service.log` |
-| xroad-jetty                       |       The application server providing the user interface and the request acceptance service.        |                                 `/var/log/xroad/jetty/` |
-| xroad-signer                      |                                The service that manages key settings.                                |                             `/var/log/xroad/signer.log` |
-| nginx                             | The Web server that distributes configuration and implements the TLS protocol in the user interface. |                                       `/var/log/nginx/` |
+| Service                           | Purpose                                                                                             |                                                     Log |
+|-----------------------------------|-----------------------------------------------------------------------------------------------------|--------------------------------------------------------:|
+| xroad-center                      | X-Road Central Server                                                                               |        `/var/log/xroad/centralserver-admin-service.log` |
+| xroad-center-registration-service | X-Road Central Server Registration Service                                                          | `/var/log/xroad/centralserver-registration-service.log` |
+| xroad-center-management-service   | X-Road Central Server Management Service                                                            |   `/var/log/xroad/centralserver-management-service.log` |
+| xroad-jetty                       | The application server providing the user interface and the request acceptance service              |                                 `/var/log/xroad/jetty/` |
+| xroad-signer                      | The service that manages key settings                                                               |                             `/var/log/xroad/signer.log` |
+| nginx                             | The Web server that distributes configuration and implements the TLS protocol in the user interface |                                       `/var/log/nginx/` |
 
 System services can be managed using the systemd facility.
 To start a service, issue the following command as a root user:
```

Comment:
The below command should be updated:

> To start a service, issue the following command as a root user:
>
> `service <service> start`
>
> To stop a service, enter:
>
> `service <service> stop`

Use `systemctl` instead of `service`:

> `systemctl start <service>`
>
> To stop a service, enter:
>
> `systemctl stop <service>`